corehq/motech/fhir/utils.py

```diff
@@ -53,3 +56,51 @@ def update_fhir_resource_property(case_property, fhir_resource_type, fhir_resour
                                                       resource_type=fhir_resource_type)
         fhir_resource_prop.jsonpath = fhir_resource_prop_path
         fhir_resource_prop.save()
+
+
+def build_capability_statement(domain, fhir_version):
```

Comment:
I am going to use the `domain` param later as the token and authorize urls are going to be domain specific.